plugins/producer_plugin/include/eosio/producer_plugin/snapshot_scheduler.hpp

```diff
@@ -197,8 +197,8 @@ class snapshot_scheduler {
                   });
                }
             } else {     
-               // should not happen, but if happens - not a terminal error          
-               elog( "unable to finalize pending snapshot request: ${details}", ("details", srid) );
+               // this could happen when request is already "expired" so there is no value in reporting it          
+               // dlog( "unable to finalize pending snapshot request: ${details}", ("details", srid) );
```

Comment:
We normally do not leave commented out code behind. We remove them.

Reply:
lets remove it! thank you!